### ISSUE TITLE
Following banks ifsc not available in bankname.json

### ISSUE THREAD

**Issue by whfamily2006** [razorpay/ifsc#50]
1. Bank   Internasional Indonesia
2. Capital   Local Area Bank Ltd
3. Development   Credit Bank Ltd
4. Oman   International Bank SAOG
5. Parsik   Janata Sahakari Bank Ltd
6. Ratnakar   Bank Limited
7. The   Lakshmi Vilas Bank Ltd
8. UBS AG
9. Westpac   Banking Corporation

**Comment by captn3m0** [comment]
- Capital Local Area Bank is now a Payments Bank and called Capital Small Finance Bank
- Correct spelling for Lakshmi Vilas Bank is "Laxmi Vilas Bank"
- Development Credit Bank is now called DCB Bank only
- Oman International was merged with HSBC and is called "HSBC Bank Oman S.A.O.G"
- Parsik Janta's correct name is named as "Gopinath Patil Parsik Janata Sahakari Bank" in the sheets. I'll see if we need to shorten this to `GP`
- Ratnakar Bank calls itself RBL
- UBS shut down in India in 2013: https://timesofindia.indiatimes.com/business/india-business/UBS-to-shut-India-banking-unit/articleshow/20709786.cms
- Westpac is in the file: https://github.com/razorpay/ifsc/blame/master/src/banknames.json#L1329

I think IBBK had a similar merger, but need to double check that.

**Comment by whfamily2006** [comment]
Thank you @captn3m0

**Comment by captn3m0** [comment]
`IBBK` is the same case as Oman. RBI has two banks in this list: https://www.rbi.org.in/Scripts/bs_viewcontent.aspx?Id=2009

- [Bank Internasional Indonesia](http://rbidocs.rbi.org.in/rdocs/Content/DOCs/IFCB2009_152.xls)
- [PT Bank Maybank Indonesia TBK](http://rbidocs.rbi.org.in/rdocs/Content/DOCs/IFCB2009_190.xls)

And both of them have a single entry (`IBBK0000001`) with a different bank name. Thankfully, rest of the details are same, unlike the Oman case (which has a different branch name I think)

We have to pick one, we just picked the second instead of the first.
